suse_sphinx_theme/titlerow.html

```diff
@@ -8,10 +8,10 @@ <h2>OpenStack API Documentation</h2>
   </div>
   <div class="docs-actions">
     {% if prev %}
-    <a href="{{ prev.link|e }}"><i class="fa fa-angle-double-left" data-toggle="tooltip" data-placement="top" title="Previous: {{ prev.title|striptags }}"></i></a>
+    <a href="{{ prev.link|e }}"><i class="material-icons" data-toggle="tooltip" data-placement="top" title="Previous: {{ prev.title|striptags }}">navigate_before</i></a>
```

Comment:
Can I have a screenshoot of the current implementation of this, please? I'm having some trouble to get it running locally.

Reply:
I'm not sure, are you asking for a screenshot of the current icons? I sent an email containing one.

Reply:
![nova_docs_updated_suse_theme](https://user-images.githubusercontent.com/23247873/51863956-46b45280-22f7-11e9-9925-6805c085bb1e.png)
![nova_docs_updated_suse_theme_footer](https://user-images.githubusercontent.com/23247873/51863957-46b45280-22f7-11e9-9b03-699dfc38de9e.png)

Reply:
screenshots added, note these are 2 different screenshots, not 1 with a weird offset in the middle!
(looks funny on my browser)

Reply:
The way we have it in EOS, is a content indicator.
![image](https://user-images.githubusercontent.com/940170/51870280-6711fa80-2353-11e9-98f5-3f43f1b72d3f.png)
You can tell that there's more content on the right, once you reach the limit it changes:
![image](https://user-images.githubusercontent.com/940170/51870334-89a41380-2353-11e9-9c2d-1a3f7e26a4f1.png)

I understand that it's for moving between pages, but I think it would be a clear indicator for first and last page =)

Reply:
Oh I see!
None of the examples are defined yet, personal taste can work, both make sense =).